shared/src/commonMain/kotlin/xyz/alaniz/aaron/lightsaber/ui/settings/SettingsUi.kt

```diff
@@ -193,11 +212,28 @@ fun LightsaberBladeColorDropdownMenu(
 
 @Composable
 private fun LightsaberCircle(color: Color, modifier: Modifier = Modifier) {
+    val circleModifier = if (color == LightsaberCustom) {
+        modifier
+            .size(32.dp)
+            .clip(CircleShape)
+            .background(
+                Brush.sweepGradient(
+                    listOf(
+                        Color.Red,
+                        Color.Green,
+                        Color.Blue,
+                        Color.Red
+                    )
+                )
+            )
+    } else {
+        modifier
+    }
     Surface(
         shape = CircleShape,
         color = Color.White,
-        border = BorderStroke(4.dp, color),
-        modifier = modifier.size(32.dp).blur(4.dp).clip(CircleShape)
+        border = if (color != LightsaberCustom) BorderStroke(4.dp, color) else null,
```

Comment:
With this approach, the user cannot see their custom color. Can you change such that the user can see their custom color?